### PR TITLE
RHEL guest-os configuration updates

### PR DESCRIPTION
**Pull request by clebergnu**
Two set of changes:
 * Cleanup of RHEL 3 and 4
 * Addition of RHEL 6.9, including:
    * regular and "perf" kickstart files
    * i386, x86_64 and i386 configs

### CODE REVIEW

File: shared/cfg/guest-os/Linux/RHEL/6.9/ppc64.cfg [hunk 1]
```diff
@@ -0,0 +1,16 @@
+- ppc64:
+    del grub_file
+    vm_arch_name = ppc64
+    image_name += -ppc64
```
Comment (luckyh) [inline shared/cfg/guest-os/Linux/RHEL/6.9/ppc64.cfg]: Better to add `guest_name += ppc64` like what x86 does.

**Comment by yashkmankad** [comment]
I agree, the RHEL-3 and RHEL-4 kickstart files need to be removed.

Reviewed-by: Yash Mankad <ymankad@redhat.com>

File: shared/cfg/guest-os/Linux/RHEL/6.9/ppc64.cfg (hunk 1 above)
Comment (clebergnu) [inline shared/cfg/guest-os/Linux/RHEL/6.9/ppc64.cfg]: @luckyh it makes sense to keep the ppc64 consistent with the x86_64 configuration, but it doesn't seem consistent with *other* guest OS definitions.  The only reference that I see to the `guest_name` parameter is here:

https://github.com/avocado-framework/avocado-vt/blob/master/virttest/utils_test/libvirt.py#L1806

And I don't even see that function being used on tp-qemu.  I'll actually remove this on v2.

**Comment by clebergnu** [comment]
Closing in favor of #1061

**Pull request closed without merging**